lucene/core/src/java/org/apache/lucene/store/Directory.java

```diff
@@ -165,4 +165,13 @@ public void copyFrom(Directory from, String src, String dest, IOContext context)
    * @throws AlreadyClosedException if this Directory is closed
    */
   protected void ensureOpen() throws AlreadyClosedException {}
+  
```

Comment:
I think we should avoid changing any lucene classes for the moment - fileModified() can probably stay where it is?